UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Startup.cs

```diff
@@ -51,11 +84,69 @@ protected IConfigurationRoot BuildConfiguration(IWebHostEnvironment hostingEnvir
                 .AddJsonFile("appsettings.json", false, true);                
 
             builder.AddEnvironmentVariables();
+            var tempConfig = builder.Build();
+            string kvServiceUri = tempConfig["KeyVaultSettings:ServiceUri"];
+
+            if (!string.IsNullOrWhiteSpace(kvServiceUri))
+            {
+                builder.AddAzureKeyVault(new Uri(kvServiceUri),
+                    new DefaultAzureCredential(new DefaultAzureCredentialOptions { ManagedIdentityClientId = tempConfig["ESSManagedIdentity:ClientId"] }));
+            }
```

Comment:
No such configuration added 'ESSManagedIdentity:ClientId'. Please update with correct configuration, if required.

Reply:
Yes. I missed that. Thanks for highlighting that.
Made changes accordingly.